.github/vale/bin/single-line-check

```diff
@@ -4,14 +4,14 @@
 
 RC=0
 
-NEWLINE_CHECK=$(grep -v 'alt=' $(git diff --diff-filter=d --name-only $1 | grep '\.md$') | grep -v 'md:#' | grep -v '!\[' | grep -v '//' | grep -v '|' | grep -v '#' | grep '[a-z]\. [A-Z]')
+NEWLINE_CHECK=$(grep -v 'alt=' $(git diff --diff-filter=d --name-only $1 | grep '\.md$') | grep -v 'md:#' | grep -v '!\[' | grep -v '//' | grep -v '|' | grep -v '#' | grep '[a-z]\. ')
```

Comment:
```suggestion
NEWLINE_CHECK=$(grep -v 'alt=' $(git diff --diff-filter=d --name-only $1 | grep '\.md$') | grep -v 'md:#' | grep -v '!\[' | grep -v '//' | grep -v '|' | grep -v '#' | grep '\. ')
```
[opt] I don't think the sentence needs to end in a lower case letter does it? What if it ends in a symbol (like in a link or something) or abbreviation or some such?